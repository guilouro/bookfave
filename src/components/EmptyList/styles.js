import styled from 'styled-components'
import { colors } from '../../utils.styles'

export const Container = styled.div`
  width: 100%;
  height: 100%;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  color: ${colors.secondary};
  padding-top: 50px;
`
