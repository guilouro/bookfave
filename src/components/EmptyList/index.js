import React from 'react'
import * as S from './styles'
import Icon from '../Icons'

const EmptyList = () => (
  <S.Container>
    <Icon name="Empty" size="50px" />
    <p>We haven't any favorite yet</p>
  </S.Container>
)

export default EmptyList
