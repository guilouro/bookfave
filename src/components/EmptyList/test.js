import React from 'react'
import EmptyList from '.'
import { render } from 'react-testing-library'

describe('EmptyList', () => {
  it('Should render correctly', () => {
    const { container } = render(<EmptyList />)
    expect(container).toMatchSnapshot()
  })
})
