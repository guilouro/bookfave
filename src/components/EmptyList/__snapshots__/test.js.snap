// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyList Should render correctly 1`] = `
.c0 {
  width: 100%;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #b0b9bf;
  padding-top: 50px;
}

<div
  class="c0"
>
  <svg
    fill="currentColor"
    height="50px"
    viewBox="0 0 475 475.884"
    width="50px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M473.738 328.543A23.656 23.656 0 0 0 450.922 312h-10.48V56a8 8 0 0 0-8-8h-112V8a8 8 0 0 0-8-8h-304a8 8 0 0 0-8 8v416a8 8 0 0 0 8 8h264.32l-4 12.336a23.993 23.993 0 0 0 8.695 26.879 24 24 0 0 0 28.25-.016l41.934-30.543a8.06 8.06 0 0 1 9.511 0l42.012 30.559a23.655 23.655 0 0 0 28.223 0 23.66 23.66 0 0 0 8.718-26.871l-16.062-49.489a7.915 7.915 0 0 1 2.934-8.863l42.09-30.558a23.71 23.71 0 0 0 8.671-26.891zM424.441 64v248h-25.52a7.941 7.941 0 0 1-2.472-.406 8.15 8.15 0 0 1-5.117-5.106L375.25 257.04a24.002 24.002 0 0 0-45.602 0l-9.207 28.266V208h64a8 8 0 0 0 0-16h-64v-32h64a8 8 0 0 0 0-16h-64v-32h64a8 8 0 0 0 0-16h-64V64zm-408 352V16h288v296h-50.48a24.004 24.004 0 0 0-22.879 16.566 24.01 24.01 0 0 0 8.734 26.868l42.075 30.55a7.927 7.927 0 0 1 2.945 8.903L277.969 416zm439.2-73.504l-42.079 30.559a23.87 23.87 0 0 0-8.738 26.777l16.082 49.441a7.999 7.999 0 0 1-12.32 8.973l-42.04-30.582a24.2 24.2 0 0 0-28.265 0l-41.992 30.559a7.999 7.999 0 1 1-12.328-8.965l16.074-49.418a23.883 23.883 0 0 0-8.746-26.817l-42.047-30.52A8.002 8.002 0 0 1 253.961 328h52a22.367 22.367 0 0 0 9.648-2.098l.153-.086.144-.046c.114-.051.192-.145.305-.204a23.754 23.754 0 0 0 12.559-14.132L344.852 262a8.003 8.003 0 0 1 7.597-5.504 8.004 8.004 0 0 1 7.602 5.504l16.086 49.473A24.006 24.006 0 0 0 398.922 328h52a7.995 7.995 0 0 1 7.664 5.52 7.994 7.994 0 0 1-2.938 8.976zm0 0"
    />
    <path
      d="M72.441 80h176a8 8 0 0 0 0-16h-176a8 8 0 0 0 0 16zm0 0M72.441 144h176a8 8 0 0 0 0-16h-176a8 8 0 0 0 0 16zm0 0M72.441 208h176a8 8 0 0 0 0-16h-176a8 8 0 0 0 0 16zm0 0M256.441 264a8 8 0 0 0-8-8h-176a8 8 0 0 0 0 16h176a8 8 0 0 0 8-8zm0 0"
    />
  </svg>
  <p>
    We don't have any favorite
  </p>
</div>
`;
